CHANGELOG.md

```diff
@@ -1,3 +1,284 @@
+- machine-controller is updated to v1.41.0 ([#8590](https://github.com/kubermatic/kubermatic/issues/8590))
+- machine-controller updated to v1.41.1  
+  operating-system-manager updated to v0.3.6 ([#8693](https://github.com/kubermatic/kubermatic/issues/8693))
+- Update machine-controller to v1.42.0 ([#8731](https://github.com/kubermatic/kubermatic/issues/8731))
+- Update machine controller to 1.42.1 ([#8757](https://github.com/kubermatic/kubermatic/issues/8757))
+- Removed support for Kubernetes 1.19 ([#8167](https://github.com/kubermatic/kubermatic/issues/8167))
+- Add support for Kubernetes 1.22.5, 1.21.8, and 1.20.14  
+  Automatically upgrade clusters running Kubernetes 1.21 to 1.21.8 to include fixes for CVE-2021-44716 and CVE-2021-44717  
+  Automatically upgrade clusters running Kubernetes 1.22 to 1.22.5 to include fixes for CVE-2021-44716 and CVE-2021-44717  
+  Update to Go 1.17.5 ([#8472](https://github.com/kubermatic/kubermatic/issues/8472))
+- Added support for Kubernetes version v1.20.13 and automatically upgrading clusters with version < v1.20.13 (fixes CVE-2021-3711, CVE-2021-3712, CVE-2021-33910)  
+  Added support for Kubernetes version v1.21.7 and automatically upgrading clusters with version < v1.21.7 (fixes CVE-2021-3711, CVE-2021-3712, CVE-2021-33910)  
+  Added support for Kubernetes version v1.22.4 and automatically upgrading clusters with version < v1.22.4 (fixes CVE-2021-3711, CVE-2021-3712, CVE-2021-33910) ([#8251](https://github.com/kubermatic/kubermatic/issues/8251))
+
+
+### Dashboard
+- Allow users to select one of the two supported CNIs - Canal or Cilium ([#3709](https://github.com/kubermatic/dashboard/issues/3709))
+- Remove type column from cluster list ([#3751](https://github.com/kubermatic/dashboard/issues/3751))
+- Add account audits support to the admin settings ([#3782](https://github.com/kubermatic/dashboard/issues/3782))
+- Allow configuring AWS AssumeRole credentials for user clusters ([#3811](https://github.com/kubermatic/dashboard/issues/3811))
+- Add Konnectivity support ([#3822](https://github.com/kubermatic/dashboard/issues/3822))
+- Add support for RHEL and Flatcar to the VSphere provider. Make RHEL subscription manager fields optional ([#3824](https://github.com/kubermatic/dashboard/issues/3824))
+- Disable dialog select form until existing template is selected ([#3836](https://github.com/kubermatic/dashboard/issues/3836))
+- Fix a bug where OPA data was not retrieved when there were no machine deployments in the cluster ([#3851](https://github.com/kubermatic/dashboard/issues/3851))
+- Add support for Openstack authentication with `project name` and  `project id`.  `tenant name`  and `tenant id`  are deprecated and have been renamed to `project name` and  `project id` in the UI ([#3854](https://github.com/kubermatic/dashboard/issues/3854))
+- Added audit policy preset picker to the wizard ([#3862](https://github.com/kubermatic/dashboard/issues/3862))
```

Comment:
All past tenses into today ("Added" to "Add").

Reply:
done :)